packages/@azure/servicebus/data-plane/test/topicFilters.spec.ts

```diff
@@ -55,15 +55,22 @@ async function beforeEachTest(): Promise<void> {
   }
 
   ns = Namespace.createFromConnectionString(process.env.SERVICEBUS_CONNECTION_STRING);
-  topicClient = getSenderClient(ns, ClientType.TopicFilterTestTopic) as TopicClient;
-  subscriptionClient = getReceiverClient(
+
+  let clients = await getSenderReceiverClients(
     ns,
+    ClientType.TopicFilterTestTopic,
     ClientType.TopicFilterTestSubscription
-  ) as SubscriptionClient;
-  defaultSubscriptionClient = getReceiverClient(
+  );
+  topicClient = clients.senderClient as TopicClient;
+  subscriptionClient = clients.receiverClient as SubscriptionClient;
+
+  clients = await getSenderReceiverClients(
     ns,
+    ClientType.TopicFilterTestTopic,
     ClientType.TopicFilterTestDefaultSubscription
-  ) as SubscriptionClient;
+  );
+  topicClient = clients.senderClient as TopicClient;
+  defaultSubscriptionClient = clients.receiverClient as SubscriptionClient;
```

Comment:
Will this work as expected?

The second time you call `getSenderReceiverClients`, wouldn't the topic and subscription created in the first call be deleted? Yes, you will end up creating the topic again in the second call, but the subscription in the first call will not exist anymore.

Reply:
Oh yeah, missed it.

Reply:
I am not sure if making that change will make any difference...

What ClientType will you send?

This is the only case where we need 2 receiver clients for the same sender client.

Reply:
There are only two tests that use a default-subscription.
For them, 
`beforeEachTest(ClientType.TopicFilterTestDefaultSubscription)` is called.
For the rest,
`beforeEachTest(ClientType.TopicFilterTestSubscription)` is called.
And `beforeEachTest` - function would be modified accordingly.

Reply:
ok